### DISCUSSION TITLE
Champions Friday August 5, 2022 text standup.

### DISCUSSION THREAD

**Discussion by fire** [omigroup/omigroup#266] [General]
For Friday's text standup, over the last week from July 29th to August 5th, 2022.

I (fire), Lyuma, and Tokage worked on the Godot Engine skeletal retargeting system with automatic bone mapping. Lyuma completed a prototype of vrm import using this retargeting system.

Because of @mrmetaverse (Jesse, AMA) 's nudge and @humbletim 's work, we completed a Virtual Market (Hikky) VR convention booth of OMI. Let me know of any links or booth.pm items to be associated. Also, we want to present a walkthrough when the event is live. Probably want to do a retrospective meeting on the entire process.

Godot Engine as a Metaverse game engine version 4 feature froze as of yesterday, August 3rd, 2022. 

What is iFire doing for the next champions Friday?

The following two weeks are the ACM Siggraph scientific conference for computer graphics locally in Vancouver.

Godot Engine 4 expects to have a beta in 4-5 weeks. Will need to merge all of outstanding work in progress. This includes gltf extension import.

What is iFire's blockers?

Wish to have a slide deck of recent OMI work to present at gatherings before next Monday August 8th, 2022 conference. (edited)

**Comment by mrmetaverse** [comment]
## This past week: 
- Held a backlog refinement special session on Tuesday. 
- Coordinated with Humbletim on a new readme. 
- Facilitated the Wednesday meeting. Took notes. 
- Created the graphic for the topic. Made announcement, and engaged on Twitter. 
-  Community engagement, and as always, encouraging new people to attend the Wednesday meeting. 

## Intentions for this next week: 
- Demos this last week were a secondary priority. There have been some items to discuss, and I think that is always a nice way of shaking things up while staying productive. I haven't even identified a theme for next week so maybe we skip a theme, and have an open conversation. Then resume with a theme we select as a group for next-next time. 
- I like the way humbletim organized the new readme on our omigroup/omigroup Github repo. Particularly the way that working groups are defined. We have some proposals in the air that are worth reviewing Wednesday. The UX research group was recently shaken-up with a new post and update regarding the delay. I'd like to think through how we can set up this working group for success before, or after the vote. I also think we could think through ways of getting involed in working groups, and requesting help (through github backlog, or some augmented approach)
- Happy to facilitated Wednesday again, if the group is still ok with how I have been doing it. 


## Blockers
I am unsure of how Mee6 works,  and would like to see us close out the [Milestone: Selecting our Bots](https://github.com/omigroup/omigroup/milestone/3)

## Balance 
This week I would say I feel at a 8 (8/10).
